numba/tests/npyufunc/test_gufunc.py

```diff
@@ -62,6 +62,24 @@ def test_ufunc_like(self):
 
         np.testing.assert_equal(out, x * x + x)
 
+    def test_axis(self):
+        @guvectorize(["f8[:],f8[:]"], "(n)->(n)")
```

Comment:
Maybe include a comment here pointing to: https://github.com/numba/numba/issues/6773 -- IIRC this is done quite often for tests that expose specific bugs in the Numba code-base.

Reply:
will do

Reply:
done in https://github.com/numba/numba/pull/6786/commits/20c272d5d387c477197cd3bee18abc6adc445a32